### ISSUE TITLE
More details about the training?

### ISSUE THREAD

**Issue by asheroin** [sniklaus/softmax-splatting#3]
Hi, I'm trying to re-produce the result based on the DAIN repo but fails.

I haved tried mostly to match the setting which had be provided in your paper excepted the image synthesis network, including the feature pyramid etractor and the softmax splatting using the predefined brightness constancy with a scale of -20. I did not finetune the PWCNet.

However, the PSNR in Vimeo90k test set would never raise up to 33.5 after 30k~ traning samples, which could easily raise to 34 when not using SoftSplat. Further more, I have removed the depth and kernel part in DAIN and only use the FlowProjection layer, which is similar to your Average Splating, with the same synthesis network in my SoftSplat re-producing before switch to it.

I'm afraid that the problem may not be the synthesis network since it could raise the a higher PSNR when using a simplify framework. I'm afraid that there is still some importance setting for the training proceduce, such as the training epoch and the number of traning smaples for every epoch. Besides, the data aug and the total training time would be useful information.

**Comment by sniklaus** [comment]
Would you mind elaborating what you are trying to reproduce? I am afraid that we do not use DAIN in our paper, we only compare to it in the evaluation.

**Comment by asheroin** [comment]
> Would you mind elaborating what you are trying to reproduce? I am afraid that we do not use DAIN in our paper, we only compare to it in the evaluation.


Reproduce the softsplat's result by reusing the code in DAIN's repo, like the dataloader for vimeo90k , the training procedure and the network interface. Some modifications have been conducted for my environment but they are mostly the same.


Sorry for my confusing description and poor English.

**Comment by sniklaus** [comment]
I am afraid that my time is limited and I do unfortunately not have the capacity to help you integrate softmax splatting into your own work, my apologies.

**Comment by asheroin** [comment]
> I am afraid that my time is limited and I do unfortunately not have the capacity to help you integrate softmax splatting into your own work, my apologies.

A rough information may helps for reproducing, such as the total trainning days at certain batch size. It might be difficult for you to share all about the training details, but any information would helps.

**Comment by sniklaus** [comment]
With exception of the training data, we adobe the training from: https://arxiv.org/abs/1803.10967

In summary, we used AdaMax with a learning rate of 0.001 and 8 samples per batch. It has been a while since I ran the training, but it was like a day or two for 50 epochs on an oldish Nvidia Titan X.
